### PR TITLE
python3Packages.jupyter-server-mathjax, python3Packages.nbdime: fix build on darwin

### PR DESCRIPTION
**Pull request by smancill**
###### Description of changes

Enable access to local network on darwin for tests that need it.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [X] x86_64-darwin
  - [ ] aarch64-darwin
- [X] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/nbdime/default.nix [hunk 1]
```diff
@@ -33,6 +33,24 @@ buildPythonPackage rec {
...
+    jupyter-server-mathjax
+    nbformat
+    colorama
+    pygments
+    tornado
+    requests
+    GitPython
+    notebook
+    jinja2
+    ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/nbdime/default.nix:52]:
```suggestion
  ];
```

File: pkgs/development/python-modules/nbdime/default.nix [hunk 2]
```diff
@@ -48,28 +66,16 @@ buildPythonPackage rec {
     "test_apply_filter_no_repo"
     "test_diff_api_checkpoint"
     "test_filter_cmd_invalid_filter"
-    "test_inline_merge"
+    "test_inline_merge_source_add"
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/nbdime/default.nix:69]: We should add a comment why we disabled the tests

**Comment by smancill** [comment]
I ran nixpkgs-fmt on nbdime now when refactoring the derivation.

File: pkgs/development/python-modules/nbdime/default.nix [hunk 3]
```diff
@@ -33,6 +36,24 @@ buildPythonPackage rec {
...
+    tornado
+    requests
+    GitPython
+    notebook
+    jinja2
+  ];
+
   checkInputs = [
     hypothesis
     pytest-cov
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/nbdime/default.nix:59]:
```suggestion
```
also we don't want to generate coverage

**Pull request merged**